src/tailwind.css

```
@@ -1,176 +1,307 @@
+ * consistency across themes. Only introduce a new theme variable when it is highly reusable and
+ * carries significant semantic meaning (e.g. a distinct UI role that cannot be expressed with the
+ * existing tokens). One-off colors, feature-specific shades, or near-duplicates of existing
+ * tokens do not qualify. Extensions have a little more freedom to experiment with new theme
+ * variables than the core platform does, but they should still be added based on these criteria.
+ *
+ * If you do decide to add a new semantic color, you MUST receive direct approval from UX before
+ * merging.
+ */
+
+/* CUSTOM: Added `tw:` prefix */
+@import 'tailwindcss' prefix(tw);
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)